src/vs/workbench/contrib/debug/browser/debugANSIHandling.ts

```diff
@@ -96,23 +86,122 @@ export function handleANSIOutput(text: string, linkDetector: LinkDetector): HTML
 			} else {
 				currentPos = startPos;
 			}
-
 		}
 
 		if (sequenceFound === false) {
 			buffer += text.charAt(currentPos);
 			currentPos++;
 		}
-
 	}
 
 	// Flush remaining text buffer if not empty.
 	if (buffer) {
-		appendStylizedStringToContainer(root, buffer, styleNames, linkDetector);
+		appendStylizedStringToContainer(root, buffer, styleNames, linkDetector, customFgColor, customBgColor);
 	}
 
 	return root;
 
+	/**
+	 * Change the foreground or background color by clearing the current color
+	 * and adding the new one.
+	 * @param newClass If string or number, new class will be
+	 *  `code-(foreground or background)-newClass`. If `null`, no new class
+	 * 	will be added.
+	 * @param isForeground If `true`, will change the foreground color, if
+	 * 	`false`, will change the background color.
+	 * @param customColor If provided, this custom color will be used instead of
+	 * 	a class-defined color.
+	 */
+	function changeColor(newClass: string | number | null, isForeground: boolean, customColor?: RGBColor | null): void {
```

Comment:
Instead of the isForeground: boolean. I think it would be clearer to just have colorType: 'foreground' | 'background' in all methods
That way it would be more clear and you would not need all these comments which explan the menaing of isForeground

Reply:
Do you think an `enum` would be better than strings here?

Reply:
I would just put two string values, typescript will declare that type to be only one of those two values

Reply:
Fixed in 468260e.